source/common/grpc/rpc_channel_impl.cc

```diff
@@ -124,9 +124,6 @@ void RpcChannelImpl::onFailure(Http::AsyncClient::FailureReason reason) {
   case Http::AsyncClient::FailureReason::Reset:
```

Comment:
given that this is the only possibility, maybe the parameter can be removed completely?  Does FailureReason even need to exist now?